motor/core.py

```diff
@@ -2034,3 +2034,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     async def get_keys(self):
         cursor_class = create_class_with_framework(AgnosticCursor, self._framework, self.__module__)
         return cursor_class(self.delegate.get_keys(), self)
+
+    async def create_encrypted_collection(self, *args, **kwargs):
+        collection_class = create_class_with_framework(
+            AgnosticCollection, self._framework, self.__module__
+        )
+        coll, ef = self.delegate.create_encrypted_collection(*args, **kwargs)
```

Comment:
This is directly calling the synchronous pymongo method which will block the event loop. What we need to do here is wait for the create_encrypted_collection call to be run in the background so that we don't block the loop. See AgnosticChangeStream.try_next for an example.

Reply:
Done.